test-karma/client/js/client.spec.js

```diff
@@ -50,7 +50,7 @@ describe('GME client', function () {
         it('should have public functions', function () {
             var client = new Client(gmeConfig);
 
-            expect(client.hasOwnProperty('gmeConfig')).to.equal(true, 'gmeConfig');
+            expect(Object.hasOwn(client, 'gmeConfig')).to.equal(true, 'gmeConfig');
```

Comment:
There is something satisfying about all these lines keeping their original length :nerd_face:

Reply:
👯‍♀️